File: Programa7.java
package com.mycompany.p7;
import java.util.Scanner;

public class p7 {

    public static void main(String[] args) {
      Scanner sc = new Scanner (System.in);
      int T;
      System.out.println("introduzca la temperatura :"); 
      T =sc.nextInt(); 
      if(T>=27){ 
          System.out.println("hace calor!"); 
      }else if (T>20 && T <=27){
          System.out.println("el clima es agradable");
      } else if ( T < 20){
                  System.out.println("El clima es frio");
      }
    }
}
  

File: programa7.cs
using System;

namespace programa7 // Note: actual namespace depends on the project name.
{
    internal class Temperatura
    {
        static void Main(string[] args)
        {
            double T;
            Console.WriteLine("¿Qué temperatura hay hoy?");
            T = double.Parse(Console.ReadLine());

            if (T >= 27)
            {
                Console.WriteLine("¡Hace calor!");
            }
            else if (T < 27 && T >= 20)
            {
                Console.WriteLine("¡Clima agradable!");
            }
            else
            {
                Console.WriteLine("¡Clima fresco");

            }
        }
    }
}